contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -0,0 +1,119 @@
+# Scoped Kubelet API Access
+
+Author: Jordan Liggitt (jliggitt@redhat.com)
+
+## Overview
+
+Kubelets are responsible for:
+* creating and updating status of their Node API object
+* updating status of Pod API objects bound to their node
+* creating/deleting "mirror pod" API objects for statically-defined pods running on their node
+* reading Secret and ConfigMap objects referenced by pod specs bound to their node
```

Comment:
Also, I think kubelets rely on an (implicit?) list of all Pods in the system via a watch to pods?  If so, then Kubelets have an additional requirement to
* get a list of Pods that are scheduled to that kubelet.

That is, the problem of allowing *read* access to a specific pod is different from a list query to find all visible-to-me pods?